### ISSUE TITLE
prefer_zero_over_explicit_init Not working for UIEdgeInsets

### ISSUE THREAD

**Issue by skatolyk** [realm/SwiftLint#3986]
### New Issue Checklist

- [ ] Handle new UIEdgeInsets.zero case

### Describe the bug

The use of prefer_zero_over_explicit_init doesn't cover UIEdgeInsets.zero case.

For example we have next code

```swift
extension UIApplication {
    static var safeAreaInsets: UIEdgeInsets {
        UIWindow.key?.safeAreaInsets ?? UIEdgeInsets(top: 0, left: 0, bottom: 0, right: 0) // Here should be swiftlint prefer_zero_over_explicit_init  warning 
    }
}
```

**Comment by marcelofabri** [comment]
PRs are welcome!
